### ISSUE TITLE
Output from cascading and PD-Net

### ISSUE THREAD

**Issue by sunjoy158** [zaccharieramzi/fastmri-reproducible-benchmark#128]
Sorry to bother, I have a questions about the models inputs and outputs.

When I was running the cascading model and PDnet model, I found that the input was (None, 640, None, 1) and the final output was (None, None, None, 1). Could you help me explain what was the actual input and output and show the final figure from those two models?

**Comment by zaccharieramzi** [comment]
Hello @sunjoy158 ,

You are not bothering at all. Please don't hesitate to ask questions especially here, since others may have the same.

So what you are referring to here are the shape specifications for (part of) the input and the output of the functional models.
For example the inputs for a PDNet are the k-space and the under-sampling mask.
The k-space has a batch dimension (i.e. the slice dimension) which is not constrained since we don't know in advance the number of slices, a readout dimension of 640 (here I was only considering single-coil knee data), a phase dimension which is unconstrained and it has one channel exactly.
For unconstrained dimensions, the shape specification is None, which for the k-space gives `(None, 640, None, 1)`.

The shape specification of the output is computed automatically by `tf.keras`, but you can see it corresponds to what you might expect from a greyscale image: slice x height x width x channels -> `(None, None, None, 1)`.

I am not I understand what you mean by final figure. Could you detail this question?

**Comment by zaccharieramzi** [comment]
@sunjoy158  any news on this?

**Comment by zaccharieramzi** [comment]
@sunjoy158 I am closing this due to lack of response, but feel free to reopen if you have any further questions with which I could help.
